Products/ZenModel/actions.py

```diff
@@ -394,6 +396,37 @@ def _encodeBody(self, body):
             plain_body = MIMEText(body.decode('ascii', 'ignore'))
         return plain_body
 
+    def _evtTimeToUserTz(self, body, zone_name):
```

Comment:
I was referring to a standard notification template, that is why it locates a single times, and yes it will only change one timestamp, which will be a problem.